### ISSUE TITLE
Dropped limbs do not update clothes overlays right

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#30873]
(WEB REPORT BY: kanef REMOTE: 206.221.180.138:7777)
# Revision
bea1b408876f3a63a4badea318d57ddb62cc371d

# Description
Dropped limbs do not remove holes in clothes overlays that show limbs
# Steps to Reproduce
* Surgery table
* Saw limbs off while clothed
* Turn sprite to side
# What you Expected
A clean view of the clothes from the side with no hole in the sprite for arms
# What Actually Happened
Hole in the sprite still there
# Ideal solution
Make clothing limbs seperate overlays
[bugfix][needspritework]

**Comment by DeityLink** [comment]
I think you really don't realize the scale of what you're talking about. At the very least that means either two sprites for each clothes, or one sprite and a mask icon_state for the hands (some of which could be shared by a few clothes I guess). It's not undoable but basically requires redoing every uniform/suit in the game. Oh and gloves too I guess to account for people who lose a hand.

**Comment by DeityLink** [comment]
Now that I've described the potential solution it actually seems somewhat manageable.

**Comment by ghost** [comment]
yeah also i really wanted to do this so that limbs could have individual transforms for more fun animations with them that dont require a ton of spriting, me testing this is how i found this out. also youre probably gonna have to update the inhand icons for every single item in the game so have fun. OR we could scrap them and just make them transforms, ugly solution probably but it mirrors how we gutted all the lying sprites for everything

**Comment by kane-f** [comment]
also youre probably gonna have to update the inhand icons for every single item in the game so have fun

**Comment by kane-f** [comment]
OR we could scrap them and just make them transforms, ugly solution probably but it mirrors how we gutted all the lying sprites for everything

**Comment by DamianX** [comment]
#26109
#22223

**Comment by DeityLink** [comment]
Why the inhand too? If a hand is missing they shouldn't appear to carry anything in that hand anyway

**Comment by ghost** [comment]
> Why the inhand too? If a hand is missing they shouldn't appear to carry anything in that hand anyway

fair point
